lib/std/Build/Step/ConfigHeader.zig

```diff
@@ -553,6 +553,12 @@ fn replace_variables(
                     allocator.free(content_buf);
                     content_buf = buf;
                 },
+                .ident => |ident| {
```

Comment:
This looks like the same body as `string`. Couldn't you just do `.string, .ident => |x| {`